packages/fxa-payments-server/src/components/NewUserEmailForm/index.test.tsx

```diff
@@ -150,26 +145,19 @@ describe('NewUserEmailForm test', () => {
   });
 
   it('shows no error when emails match', async () => {
-    let subject;
-    await act(async () => {
-      subject = render(
-        <WrapNewUserEmailForm accountExistsReturnValue={false} />
-      );
-      const firstEmail = subject.getByTestId('new-user-email');
-      const secondEmail = subject.getByTestId('new-user-confirm-email');
+    let subject = render(
+      <WrapNewUserEmailForm accountExistsReturnValue={false} />
+    );
+    const firstEmail = subject.getByTestId('new-user-email');
+    const secondEmail = subject.getByTestId('new-user-confirm-email');
 
-      fireEvent.change(firstEmail, { target: { value: 'valid@email.com' } });
-      fireEvent.blur(firstEmail);
-      fireEvent.change(secondEmail, { target: { value: 'valid@email.com' } });
-      fireEvent.blur(secondEmail);
-    });
-    waitForElementToBeRemoved(
```

Comment:
tbh idk why this got caught up in the mix. it threw an unhandledPromiseRejection on 16. the more straightforward test works though 🤷